### PR TITLE
Homework4: Dmitry Radyukov

### PR DESCRIPTION
**Pull request by DmitryRadyukov**
add Homework2.py

### CODE REVIEW

**Comment by DmitryRadyukov** [comment]
Add Homework 2, 3, 4

File: Homework4.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+list = set("123")
+list2 = set("345")
+print(len(list | list2))
+
+# 4
+
+
+# 5 Languages
+
+stud = [{input() for j in range(int(input()))} for i in range(int(input()))]
```
Comment (belovmd) [inline Homework4.py:32]: great

**Review by belovmd — CHANGES_REQUESTED** [review]
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-python-spring/blob/master/homework_review/Homework4.md):
- 25
- 26
- 27
- 28
- 29
- 30
- 13
- 20

**Review by belovmd — COMMENTED** [review]
review issues not addressed. 
5

**Pull request closed without merging**